src/vsr/superblock.zig

```diff
@@ -1326,9 +1319,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
                         &working.vsr_state.members,
                         working.vsr_state.replica_id,
                     ) != null);
-                } else if (context.caller == .checkpoint) {
-                    superblock.free_set.checkpoint();
-                }
+                } else if (context.caller == .checkpoint) {}
```

Comment:
The free set is now delicately check-pointed in the middle of free-set-encoded checkpointing process.